org.eclipse.scanning.malcolm.core/src/org/eclipse/scanning/malcolm/core/MalcolmDevice.java

```diff
@@ -60,60 +62,45 @@
  */
 public class MalcolmDevice<M extends MalcolmModel> extends AbstractMalcolmDevice<M> {
 	
-	public static final class EpicsMalcolmModel {
-		private final IPointGenerator<?> generator;
-		private final List<String> axesToMove;
-		private final String fileDir;
-
-		public EpicsMalcolmModel(String fileDir, List<String> axesToMove,
-				IPointGenerator<?> generator) {
-			this.fileDir = fileDir;
-			this.axesToMove = axesToMove;
-			this.generator = generator;
-		}
-
-		public String getFileDir() {
-			return fileDir;
-		}
-
-		public List<String> getAxesToMove() {
-			return axesToMove;
-		}
+    // Static data
+	private static Logger logger = LoggerFactory.getLogger(MalcolmDevice.class);	
+	private static String STATE_ENDPOINT = "state";
+	private static String STATUS_ENDPOINT = "status";
+	private static String BUSY_ENDPOINT = "busy";
+	private static String CURRENT_STEP_ENDPOINT = "completedSteps";
 
-		public IPointGenerator<?> getGenerator() {
-			return generator;
-		}
-		
+	// Frequencies and Timeouts
+	// broadcast every 250 milliseconds
+	private final static long POSITION_COMPLETE_FREQ = Long.getLong("org.eclipse.scanning.malcolm.core.positionCompleteFrequency", 250); 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)